assets/config/0.5.5-coins.json

```diff
@@ -9810,5 +9810,59 @@
     "type": "UTXO",
     "active": false,
     "currently_enabled": false
+  },
+  "RUNES": {
+    "coin": "RUNES",
+    "name": "Runebase",
+    "coinpaprika_id": "runes-runebase",
+    "coingecko_id": "runebase",
+    "nomics_id": "runes-runebase",
```

Comment:
the nomics_id of RUNES is `RUNES`, see https://nomics.com/assets/runes-runebase/widget (`data-base="RUNES"`)